data-client/src/main/java/org/wikipedia/dataclient/mwapi/MwQueryPage.java

```diff
@@ -141,6 +151,27 @@ public void appendTitleFragment(@Nullable String fragment) {
         title += "#" + fragment;
     }
 
+    public boolean checkWhetherFileIsUsedInWikis() {
+        if (globalUsages == null || globalUsages.size() == 0) {
```

Comment:
Rather than adding a level, just first check `if (globalUsages != null && globalUsages.size() > 0) {` and return true if yes.